app/assets/javascripts/application.js

```diff
@@ -12,7 +12,30 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
 //= require_tree .
 //= require jquery
-//= require bootstrap
\ No newline at end of file
+//= require bootstrap
+
+//Menu Toggle Script
+$(document).ready(function() {
+  if (window.location.pathname == "/"){
+    $("#sidebar-wrapper-right").removeClass("hide");
```

Comment:
saw my comments in the previous pull request? https://github.com/wdi-hk-9/Project-PlaySpace/pull/12#discussion_r43726211